src/bindist/installdos

```diff
@@ -3,6 +3,8 @@
 SOURCE="$1"
 DRIVE_ROOT="$2"
 
+PROPRIETARY_DOS_INSTALLED_EXIT_CODE=100
```

Comment:
What exactly do you mean?
I've compiled your code and I can see
your menus perfectly (none of which make
any sense to me, I don't know what should
I choose, but they are visible).
I don't suppose you can print anything but
the strings returned from the script?